react-troubleshooter.md

```diff
@@ -4,6 +4,16 @@
 
 ### __invariant violation__:
 
+#### Unable to find element
+
+When the DOM has mutated & you try to modify props / state on a component, you will find that React will complain about not being able to find the element.
```

Comment:
is it specifically when the DOM above a component has been modified to remove the component's DOM or any DOM manipulation? DOM DOM DOM DOM...